packages/node-resolve/test/fixtures/node_modules/exports-browser/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "main": "index.cjs",
+  "module": "index.js",
+  "browser": {
+    "./require.js": "./ignored-browser.js"
```

Comment:
Since there is already a chaining test, how about making this a direct browser override test instead:

```json
{
  "browser": "./ignored-browser.js"
}
```

Where we can then verify that `"exports"` takes precedence over `"browser"` here.

Reply:
Thanks, a commit has been added to align the 'exports-browser' test to your suggestions